src/components/Calculator.js

```diff
@@ -1,40 +1,31 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import { v4 as uuidv4 } from 'uuid';
 import calculate from './logic/calculate';
 import './Calculator.css';
 
-class Calculator extends Component {
-  btnValues = [
+function Calculator() {
```

Comment:
- [ ] Instead of using `function`, Kindly use ES6 Syntax.For ex:-
```
const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);

```